history.txt

```diff
@@ -1,6 +1,27 @@
 OMERO version history
 =====================
 
+5.3.2 (May 2017)
+----------------
+
+This is a bug-fix.
+
+Improvements include:
+
+- improve populate_metadata plugin
+- fix deletion of a range of objects from CLI
+- allow to add textual annotation without namespace at import from CLI
+- allow administrator to move data between groups without being a group member from CLI
```

Comment:
administrators can now use the CLI to move data between groups without being a member of those groups